Bottom: b6de38d9b5ab917380029a2205830aad565dbaa2
Top:    0d0fc82bb233bec19437c1339f129aa7facec095
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 10:37:40 +0000

Refresh of try-from-first-principles-on

---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 1e1824d..52182c0 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -202,18 +202,14 @@ step_do_export (HID_Attr_Val * options)
         poly_DelContour (&curc);
       }
 
-#if 1
     mask_objects = object3d_from_soldermask_within_area (piece, TOP_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
 
     mask_objects = object3d_from_soldermask_within_area (piece, BOTTOM_SIDE);
     board_outline_list = g_list_concat (board_outline_list, mask_objects);
-#endif
 
-#if 1
     copper_layer_objects = object3d_from_copper_layers_within_area (piece);
     board_outline_list = g_list_concat (board_outline_list, copper_layer_objects);
-#endif
 
   } while ((piece = piece->f) != board_outline);
   poly_Free (&board_outline);
